teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, _classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] `Teacher` inherits from `Person` and the constructor extends the parent's constructor by adding `@specialization` and a parameter for it. this means that `_classroom` doesn't have to be included in the `Teacher` class